WhaleSpotting/react-app/src/components/SightingMapInfo.tsx

```diff
@@ -2,32 +2,64 @@ import React, { useEffect, useState } from "react";
 import WeatherApiModel from "../apiModels/WeatherApiModel";
 import { Chosen } from "./Map";
 import "../styles/SightingMapInfo.scss";
+import { Species, WhaleImageDictionary, WhaleVisualTextDictionary } from "../apiModels/CreateSightingApiModel";
 
 interface SightingMapInfoProps {
     chosen: Chosen | undefined;
 }
 
+interface IResponse {
+    lon: number | void,
+    lat: number | void,
+    species: Array<string>,
+}
+
 export default function SightingMapInfo({ chosen }: SightingMapInfoProps): JSX.Element {
 
     const key = process.env.REACT_APP_WEATHER_API_KEY;
 
-    const response = {
+    const [speciesData, setSpeciesData] = useState([]);
```

Comment:
With backend changes
```suggestion
    const [speciesData, setSpeciesData] = useState<Species[]>([]);
```

Reply:
Addressed